### PR TITLE
update lib core config

### PR DESCRIPTION
**Pull request by moritzraho**
## Description

see #124 

the lib core config version < 2 is using `patch-package` which is considerably increasing the size of the module see: https://github.com/adobe/aio-lib-core-config/issues/27#issuecomment-635749371

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -16,25 +16,39 @@
         "@oclif/errors": "^1.1.2",
         "debug": "^4.1.0",
         "jsonwebtoken": "^8.3.0"
+      },
+      "dependencies": {
+        "@adobe/aio-lib-core-config": {
+          "version": "1.2.8",
```
Comment (moritzraho) [inline package-lock.json:22]: @justinedelson  `@adobe/aio-cli-plugin-jwt-auth` is still dependent on config 1.x.x, and https://github.com/adobe/aio-cli-plugin-jwt-auth is archived in favor of `@adobe/aio-cli-plugin-auth` or `@adobe/aio-lib-ims` (recommended to depend on a lib and not a plugin in a plugin)

Comment (justinedelson) [reply package-lock.json:22]: @moritzraho is there a migration guide from aio-cli-plugin-jwt-auth to aio-lib-ims

**Comment by justinedelson** [comment]
This has been incorporated into #130

**Comment by justinedelson** [comment]
This was actually done a while ago and I finally did the update to lib-ims. Closing.

**Pull request closed without merging**